mozfun/event_analysis_udf/aggregate_match_strings/udf.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE FUNCTION event_analysis_udf.aggregate_match_strings(match_strings ARRAY<STRING>)
```

Comment:
These are basically internal UDFs, and there will be more. However I wanted them in one group, which is why I put them in mozfun, even though they're not really for external-user use (unless they are _very_ advanced).